prompb/io/prometheus/write/v2/codec.go

```diff
@@ -77,13 +76,13 @@ func (h Histogram) ToIntHistogram() *histogram.Histogram {
 		PositiveBuckets:  h.GetPositiveDeltas(),
 		NegativeSpans:    spansProtoToSpans(h.GetNegativeSpans()),
 		NegativeBuckets:  h.GetNegativeDeltas(),
+		CustomValues:     h.GetCustomValues(),
```

Comment:
probably nothing to be done with regards to optimization, but I did mention it in https://github.com/prometheus/prometheus/issues/13490 now